.github/workflows/ci.yml

```diff
@@ -8,81 +8,23 @@ on:
   workflow_dispatch:
 
 jobs:
-  unittest:
```

Comment:
Why are we removing unit test?
Same goes for doxygen check.

Is this intentional?

Reply:
I had removed the jobs so that I didn't run everything on every commit to the PR. I've added them back in at this point.